packages/dd-trace/src/appsec/rasp/ssrf.js

```diff
@@ -20,12 +21,12 @@ function disable () {
 function analyzeSsrf (ctx) {
   const store = storage.getStore()
   const req = store?.req
-  const url = ctx.args.uri
+  const uri = (ctx.args.options?.uri && url.format(ctx.args.options.uri)) ?? ctx.args.uri
```

Comment:
NIT: var is named `uri` but address is named `url`. I know it has changed due to requiring `url` module, but maybe we can name it `outgoingUrl` to match the address.

Reply:
yes, I had the same question, but as the origin value comes from a `.uri` property in both cases I decided to name it as `uri`
Anyway, I could require only `{ format } = require('url')` to avoid the naming conflict. I don't have any problem to change it.

Reply:
changed :stuck_out_tongue: